_posts/2022/2022-01-31-infrastructure-software-we-use-to-run-our-saas.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Software & Infrastructure we use to Run Our SaaS"
+description: "If you do a lot with a small team, you have to pay for softare"
+layout: article
+category: ruby
+image:
+  base: "2022/infrastructure-software-we-use-to-run-our-saas"
+  alt: "Frame of a building under construction"
+  credit: "Jacek Dylag"
+  source: "https://unsplash.com/photos/nhCPOp4A2Xo"
+
+---
+
+Our team at [CoverageBook](https://coveragebook.com/) (& [AnswerThePublic](https://answerthepublic.com/)) do _a lot_ with a _very_ small number of developers and designers. Over 4,000 paying customers & 600k monthly vistors (many using a free version of our product) all with a product team of five, including me.
```

Comment:
*and (not ampersand)

Reply:
*We have over 4,000..._and_..., all supported by...

Why would you not be included in that team of 5?

Reply:
```suggestion
Our team at [CoverageBook](https://coveragebook.com/) (and [AnswerThePublic](https://answerthepublic.com/)) do _a lot_ with a _very_ small number of developers and designers. Over 4,000 paying customers and 600k monthly visitors (many using a free version of our product) all with a product team of five, including me.
```

Reply:
I'm "supervising"